### PR TITLE
[MIG][16.0] product_special_type (from v10.0)

### PR DESCRIPTION
**Pull request by acsonefho**
Migration of module `product_special_type`.

No special change, only pre-commit stuff.

### CODE REVIEW

**Comment by acsonefho** [comment]
@rousseldenis Can review this please? :pray:

File: product_special_type/__manifest__.py [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
-    'version': '10.0.1.0.0',
-    'author': 'Camptocamp,Odoo Community Association (OCA)',
-    'license': 'AGPL-3',
-    'category': 'Product',
-    'development_status': 'Beta',
-    'maintainers': ['rousseldenis'],
-    'summary':
-    '''
+    "name": "Product Special Types",
+    "version": "16.0.1.0.0",
```
Comment (rousseldenis) [inline product_special_type/__manifest__.py:6]: @acsonefho Could you follow migration guide ? https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch

Version change should appear in a migration commit and not in pre-commit stuff. Thanks

**Comment by rousseldenis** [comment]
@acsonefho

**Comment by FrancoMaxime** [comment]
Hi @rousseldenis, I fixed your comment, can you review again?

**Comment by acsonefho** [comment]
@rousseldenis Can you merge this one please?

**Comment by acsonefho** [comment]
@rousseldenis Can you use your super-power please?

**Pull request closed without merging**